code/src/files/index.scss

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@500;600&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Inconsolata', -apple-system, monospace, 'Segoe UI', 'Roboto',
+    'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+* {
+  font-family: 'Inconsolata', monospace;
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+// Colors & Other general settings
+$mainRed: #ffadad;
+$lightGray: #f2f0f0;
+$darkGray: #aaaaaa;
+$blk: #000;
+
+@mixin btn {
+  border: none;
+  border-radius: 25px;
+  font-size: 16px;
+}
+
+// Add thought-section
+.thought__wrapper {
+  border: 1px solid $blk;
+  box-shadow: 7px 7px $blk;
+  width: 50vw;
+  margin: 30px auto;
+  padding: 20px 25px 5px;
+  position: relative;
+}
+
+.add-thought__input-field {
+  width: calc(50vw - 50px);
+  height: 60px;
+  font-family: 'Inconsolata', monospace;
+  padding: 10px;
+  margin: 10px auto 5px auto;
+  resize: none;
+}
+
+.add-thought__word-count {
+  font-size: 12px;
+  text-align: right;
+}
+
+.red-text {
+  color: red;
+}
+
+.add-thought__input-field,
```

Comment:
Since you are using both BEM methodology and Sass, do some research regarding ampersand keyword (`&`) and how can we use it. Here is small teaser of what we can achieve :

```
.add-thought {
    // ... styles for .add-thought element ...

    &__input-field {
        // ... styles for .add-thought__input-field element ...
    }
}
```

Reply:
Oooooohh, LOVE this. So it's nested, and then I simply add the part that's not equal to the "root"-class? So smart. Will definitely try this.

Reply:
Yes! And it even gets better - after Sass code is complied into normal CSS (because that's the only style language that browser understands), the nesting structure disappears and we have 100% flat structure :)